backend/open_webui/apps/webui/routers/auths.py

```
@@ -27,6 +26,7 @@
     WEBUI_SESSION_COOKIE_SAME_SITE,
     WEBUI_SESSION_COOKIE_SECURE,
     SRC_LOG_LEVELS,
+    WEBUI_BASE_PATH,
```

Ruff (0.8.2)

`open_webui.env.WEBUI_BASE_PATH` imported but unused

Remove unused import: `open_webui.env.WEBUI_BASE_PATH`

(F401)